src/components/Modal/DescriptionModal.jsx

```diff
@@ -0,0 +1,43 @@
+import { useNavigate } from "react-router-dom";
+import PropTypes from "prop-types";
+
+import Modal from "../shared/Modal/Modal";
+
+import CONTENTS from "../../utils/ModalContents";
+
+function DescriptionModal({ navigateNextPage, statement }) {
+  const navigate = useNavigate();
+
+  const title = CONTENTS.title[statement];
+  const nextButtonText = CONTENTS.button[statement];
+  const message = CONTENTS.message[statement];
+
+  const handleCloseButtonClick = () => {
+    if (statement === "warning") {
+      navigate("/");
+    } else {
+      navigate("/modal/warning");
+    }
+  };
+
+  return (
+    <Modal
+      title={title}
+      onCloseButtonClick={handleCloseButtonClick}
+      nextButtonText={nextButtonText}
+      onNextButtonClick={() => navigateNextPage(statement)}
+      isDisabledButton={false}
+    >
+      <div className="h-40 mb-4 overflow-y-scroll leading-7 whitespace-pre-line max-w-none">
+        <p className="mt-3 whitespace-pre-line">{message}</p>
```

Comment:
```suggestion
      <div className="h-40 mb-4 overflow-y-scroll leading-7 max-w-none">
        <p className="mt-3 whitespace-pre-line">{message}</p>
```
부모 태그인 `div` 또는 자식 태그인 `p` 태그 중 하나의 태그에만 `whitespace-pre-line` 을 사용해도 괜찮을 것 같습니다!